NewForce-Cohort-4/nutshell-the-canine-pirates
# Nutshell-Task Section

# Description
MVP on task ticket should be functional. See Testing Instructions below. You will see a green line at the top of the screen; this will be fixed by styling shortly. 

Fixes # (issue)
## Type of change
Please delete options that are not relevant.


# Testing Instructions
1. Register a new email
2. Log in with your new account
3. Click New Task Button -- a form should appear
4. Fill out the form, entering all fields. 
5. Click Save -- the task should now print to the DOM
6. To mark the task as complete and hide, click the checkbox.
7. Once checked, the task should NOT print to the DOM. 

# Checklist:
- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
